assets/patches/sidecars/node_driver_registrar.yaml

```diff
@@ -0,0 +1,28 @@
+spec:
+  template:
+    spec:
+      containers:
+        - name: csi-node-driver-registrar
+          securityContext:
+            privileged: true
+          image: ${NODE_DRIVER_REGISTRAR_IMAGE}
+          imagePullPolicy: IfNotPresent
+          args:
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=/var/lib/kubelet/plugins/${DRIVER_NAME}/csi.sock
+            - --v=${LOG_LEVEL}
+          # Empty env. instead of nil for json patch to append new env. vars there
+          env: []
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "rm -rf /registration/${DRIVER_NAME}-reg.sock /csi/csi.sock"]
```

Comment:
This pattern doesn't work for some operators, e.g. IBM VPC and manila. We could use "rm -rf ${SOCK_FILES_TO_DELETE)" and then each operator would define its own specific list of files to delete.

Reply:
This looks like a mistake in manila - node-driver-registrar should remove its own socket and not the driver's.

And [IBM VPC is odd](https://github.com/openshift/ibm-vpc-block-csi-driver-operator/blob/55390bc12acc933667dd9cf4e5459cc214cedb2a/assets/node.yaml#L70), `/registration/vpc.block.csi.ibm.io` should not exist.

Reply:
Btw, good catches for both of them! Thanks for reviewing all CSI drivers.